### DISCUSSION TITLE
Dependabot

### DISCUSSION THREAD

**Discussion by Steellgold** [community/community#29981] [Code Security] [answered]
Hi, I have this in the [Repo] repository (https://github.com/Steellgold/Portfolio) but I manually updated the things they asked for but it still shows me this, is this normal?

![image](https://user-images.githubusercontent.com/51505384/185476516-21dfce67-2077-4a10-b6e8-c2a643368e1d.png)

When I click on it, it shows me this: (for the 3)
![image](https://user-images.githubusercontent.com/51505384/185476827-4ca7710a-6f51-4e51-a8bd-05429a6a136c.png)

**Comment by aayushchugh** [comment, marked as answer]
You should enable Depend bot pull requests it will update your packages automatically.
You can do this in settings tab of your repo
![image](https://user-images.githubusercontent.com/69336518/185539880-37e4e306-671b-4ebf-bfbf-112038479d72.png)
